torch/csrc/jit/codegen/cuda/root_domain_map.cpp

```diff
@@ -936,6 +940,13 @@ void ComputeAtRootDomainMapBuilder::handle(TensorView* tv) {
       mapAllConsumers(DomainKey(td, id));
     }
   }
+  if (td->hasRFactor()) {
+    for (auto id : TensorDomain::noReductions(td->getRootDomain())) {
+      if (id->isBroadcast()) {
```

Comment:
@rdspring1 I thought broadcast relative to view was always defined in a following expression.

Reply:
Yes, we apply broadcast after the view operation.